src/tests/bulkConcepts/addBulkConcepts.test.jsx

```diff
@@ -74,27 +74,6 @@ describe('Add Bulk Concepts', () => {
     expect(wrapper.find('Loader')).toHaveLength(1);
   });
 
-  it('should render a loader while fetching Concepts Source', () => {
-    const properties = {
-      fetchSourceConcepts: jest.fn(),
-      addExistingBulkConcepts: jest.fn(),
-      fetchConceptSources: jest.fn(),
```

Comment:
What happens if we leave this test?

Reply:
It will fail because it will try to find a loader of length 1 but instead find length 0 because since we are searching from non-CIEL concepts it will search through the other sources and since that field no longer exists the test will fail.